build.py

```diff
@@ -9,17 +9,29 @@
 def create_parser():
     parser = argparse.ArgumentParser(description="Build PEP documents")
     # alternative builders:
-    parser.add_argument("-l", "--check-links", action="store_true")
-    parser.add_argument("-f", "--build-files", action="store_true")
-    parser.add_argument("-d", "--build-dirs", action="store_true")
+    parser.add_argument("-l", "--check-links", action="store_true",
+                        help='Checks validity of links within PEP sources. '
+                             'Cannot be used with "-f" or "-d".')
+    parser.add_argument("-f", "--build-files", action="store_true",
+                        help='Renders PEPs to "pep-XXXX.html" files (Default). '
```

Comment:
```suggestion
                        help='Render PEPs to "pep-NNNN.html" files (default). '
```

Fix caps, follow convention for command tense, and be consistent with the docs and general convention for representing unknown integers